wrappers/rust/icicle-core/src/curve.rs

```diff
@@ -12,6 +12,11 @@ pub trait CurveConfig: PartialEq + Copy + Clone {
     fn eq_proj(point1: *const c_void, point2: *const c_void) -> c_uint;
     fn to_affine(point: *const c_void, point_aff: *mut c_void);
 
+    type BaseField;
```

Comment:
Wondering if it's possible to constrain these associated types, after all we know that for example `Projective = Projective<BaseField, Self>`

Reply:
I tried that initially. But this is an unstable feature

Reply:
Right. Well, perhaps you could've just used `Projective<Self::BaseField, Self>` everywhere instead of `Self::Projective` though that's a bit wordy